.github/workflows/ci-renovate.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Create and commit Change Log fragment if it does not exist
         run: |
           if [[ ! -f "${{ steps.vars.outputs.FILE_NAME }}" ]]; then
-            echo "${{ github.event.pull_request.title }}" > ${{ steps.vars.outputs.FILE_NAME }}
```

Comment:
This was also a bit unsafe https://github.com/oasisprotocol/oasis-rosetta-gateway/pull/527